articles/TOC.yml

```diff
@@ -1,23 +1,21 @@
 - name: Microsoft Quantum Documentation
   href: index.yml
-- name: Overview
+- name: Get started with quantum computing
   items:
-  - name: Get started
-    href: welcome.md
-  - name: Five questions about quantum computing
-    items:
-    - name: What is quantum computing?
-      href: overview/what-is-quantum-computing.md
-    - name: What can a quantum computer do?
-      href: overview/quantum-computers.md
-    - name: Why learn quantum computing?
-      href: overview/why-learn-quantum-computing.md
-    - name: What is Q#?
-      href: overview/what-is-qsharp.md
-    - name: How to learn quantum computing?
-      href: overview/how-to-learn-quantum-computing.md
-- name: Install and update
+  - name: What is quantum computing?
+    href: overview/what-is-quantum-computing.md
+  - name: What can a quantum computer do?
+    href: overview/quantum-computers.md
+  - name: Why learn quantum computing?
+    href: overview/why-learn-quantum-computing.md
+  - name: What are Q# and QDK?
+    href: overview/what-is-qsharp.md
+  - name: How can I learn more?
+    href: overview/how-to-learn-quantum-computing.md
+- name: Get the QDK
   items:
+  - name: Get started with the QDK
+    href: welcome.md
   - name: Install the QDK
     items:
     - name: Installation overview
```

Comment:
Cannot find the installation instruction for VS and VS code anymore.

Reply:
They're specifically on the Q# + C# install page. We landed on that arrangement because the more important decision for new users will be what host language they choose, with the IDE's being secondary (and dependent on the host language)